hcl/strconv/quote.go

```diff
@@ -70,6 +71,9 @@ func Unquote(s string) (t string, err error) {
 					if q, _ := utf8.DecodeRuneInString(s); q == '"' {
 						continue
 					}
+					if q, _ := utf8.DecodeRuneInString(s); q == '\\' {
+						continue
+					}
```

Comment:
The performance is unlikely to matter but decoding utf8 runes is not exactly cheap, so perhaps we should just turn this into a `switch`?